testing/acceptance_test.py

```diff
@@ -645,6 +647,69 @@ def join_pythonpath(*dirs):
             "*1 passed*"
         ])
 
+    @pytest.mark.skipif(not hasattr(os, "symlink"), reason="requires symlinks")
```

Comment:
This assumes if the python/os supports symlinks, so does the filesystem the temporary directory is created. This is not a given, but as this is in testing only, and generally this is the case (both in CI and locally) in most workflows, I think this is good enough.

Reply:
I agree 😁